linuxone/install.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+
+echo -e '@community http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories
+
+BUILD_DEPENDS="
+build-base
+make
+cmake
+gcc
+g++
+zlib-dev
+yaml-dev
+postgresql-dev
+gdbm-dev
+readline-dev
+ncurses-dev
+libffi-dev
+curl
+openssh
+libxml2-dev
+libxslt-dev
+icu-dev
+logrotate
+py2-docutils
+go
+ruby-dev
+linux-headers
+shadow
+pkgconf"
+
+
+PACKAGE_LIST="
+ruby
+ruby-bigdecimal
```

Comment:
I think the install.sh is missing the `ruby-json` package. We need this package in order to commit changes to Gitlab. Please checkout this [commit](https://github.com/IBM/Kubernetes-container-service-GitLab-sample/commit/ff9f89c883cbac86cda608edc87953d495848c74) to see what Ryan had done.